test/function/index.js

```diff
@@ -115,25 +112,45 @@ describe('function', () => {
 								}
 							}
 
-							if (config.show || unintendedError) {
-								console.log(result.code + '\n\n\n');
-							}
-
-							if (config.warnings) {
-								if (Array.isArray(config.warnings)) {
-									compareWarnings(warnings, config.warnings);
+							return Promise.resolve()
+							.then(() => {
```

Comment:
It seems ESLint is not happy with the indentation here (`npm run lint`)

Reply:
This is my go-to style for promises to keep nesting low, but sure.

Would be nice to get the whole codebase onto async/await though to get all of this stuff cleaned up.

That would be pending https://github.com/rollup/rollup-plugin-typescript/pull/104 though unfortunately here. The TypeScript 2.0 upgrade on the plugin really seems to be stalling progress in general here for Rollup.